server/geode_server_protocols.py

```diff
@@ -31,132 +43,9 @@ def initializeProtocols(self):
         # tell the C++ web app to use no encoding. ParaViewWebPublishImageDelivery must be set to decode=False to match.
         self.getApplication().SetImageEncoding(0)
 
-        print(OpenGeodeServerProtocol.modules)
-        for module_name in OpenGeodeServerProtocol.modules:
-            module = import_module(module_name)
+        print(GeodeServerProtocol.modules)
+        for module_name in GeodeServerProtocol.modules:
+            module = import_module(module_name.strip())
             for module_protocol in module.protocols:
                 self.registerVtkWebProtocol(module_protocol())
 
-    def north(self):
```

Comment:
Has this code been removed or moved?

Reply:
Removed, the marker is now send from the front end.

Reply:
Ok